modules/ROOT/pages/setting-inaccessible-fields.adoc

```diff
@@ -0,0 +1,12 @@
+= Setting Inaccessible Fields 
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Improve your Mule project running on Java 17 or later by configuring the Studio debugger to hide inaccessible fields and methods.
+
+* Hide inaccessible fields: Focus on relevant and accessible data to make the debugging process more efficient. Avoid accessing fields you can't interact with.
+* Hide public methods invoked to retrieve inaccessible field values: Maintain the security and integrity of your application by avoiding access to or modification of data through methods not intended for public use.
```

Comment:
Actually, this property is only intended to reduce the verbosity of the debugger output. When unable to access a field, the Debugger tries to call public methods that may offer the value of that field. This property is to not mention which method was used, in case the user is not interested in knowing (and reducing noise on the screen).

Reply:
OK! Thanks!